### ISSUE TITLE
macOS: incorrect SDK version returned in build_version

### ISSUE THREAD

**Issue by fanquake** [lief-project/LIEF#577]
**Describe the bug**
The macOS SDK version in the `build_version` info is incorrect when accessed directly (`binary.build_version.sdk`). However if you just print `binary.build_version`, it is correct.

**To Reproduce**
```bash
echo "int main() { return 0; }" > test.cpp
clang++ test.cpp
```
The `-platform_version` args passed to the linker are `-platform_version macos 11.0.0 11.3`.

```python
import lief

binary = lief.parse("a.out")

if binary.has_build_version:
    print(binary.build_version)
    print(binary.build_version.minos)
    print(binary.build_version.sdk)
```

```bash
python test.py

Command : BUILD_VERSION
Offset  : 268
Size    : 20
Platform: MACOS
  Min OS: 11.0.0
     SDK: 11.3.0
  LD - 650.9.0


[11, 0, 0]
[11, 0, 0] # should be [11, 3, 0]
```

**Environment**
 - macOS 11.3.1
 - Mach-O
 - LIEF commit version: `0.11.4-dd13711`
 - Clang: Apple clang version 12.0.5 (clang-1205.0.22.9)

**Comment by romainthomas** [comment]
I think this issue has been already resolved through #533

**Comment by fanquake** [comment]
> I think this issue has been already resolved through #533

Thanks, looks like it 👍 . We'll use the work around for now. Any ETA on a 0.12 release?
